### PR TITLE
manifest: sdk-zephyr: apply modem driver fix

### PR DESCRIPTION
**Pull request by tomi-font**
Fixes a reset loop.

### CODE REVIEW

**Review by MarkusLassila — APPROVED** [review]

**Review by SeppoTakalo — APPROVED** [review]

**Comment by jfischer-no** [comment]
@tomi-font for removal of DNM label you have to do 42 push-ups, @shanthanordic  or @SeppoTakalo  will supervise it.

**Comment by tomi-font** [comment]
> @tomi-font for removal of DNM label you have to do 42 push-ups, @shanthanordic or @SeppoTakalo will supervise it.

:open_mouth: And now what am I meant to do with regards to this PR? CI is passing, the commit in sdk-zephyr would need to get merged.

**Comment by jfischer-no** [comment]
> > @tomi-font for removal of DNM label you have to do 42 push-ups, @shanthanordic or @SeppoTakalo will supervise it.
> 
> 😮 And now what am I meant to do with regards to this PR? CI is passing, the commit in sdk-zephyr would need to get merged.

You are probably to late for RC2. See also https://github.com/nrfconnect/sdk-zephyr/pull/1565#issuecomment-1985269848

**Comment by tomi-font** [comment]
It would be great if this one-line fix could get merged, it has already passed the CI here (and the manifest hasn't changed since). It's kinda bad in that anything using Zephyr's cellular modem driver for the nRF91 will crash in the driver init, which will cause a reset loop.

File: west.yml [hunk 1]
```diff
@@ -61,7 +61,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 8d93a161567dcc525771f70692587e6a37947b71
+      revision: pull/1565/head
```
Comment (jfischer-no) [inline west.yml:64]: Please do not update revision here, see https://github.com/nrfconnect/sdk-nrf/pull/14408

**Review by jfischer-no — CHANGES_REQUESTED** [review]

File: west.yml (hunk 1 above)
Comment (tomi-font) [reply west.yml]: Err just did it...

Comment (tomi-font) [reply west.yml]: Can this PR just be closed?

**Comment by jfischer-no** [comment]
https://github.com/nrfconnect/sdk-nrf/pull/14430#discussion_r1517460331

**Pull request closed without merging**